### DISCUSSION TITLE
SUDO password is not accepted

### DISCUSSION THREAD

**Discussion by Milamber07** [coder/code-server#6071] [Q&A]
Hi!

Basically my SUDO password (variable PASSWORD_SUDO) gets rejected when I try to install python via terminal.
I've searched and found similar problems with hashed sudo only.
Any solution?

Below is my waiting to be sent bug report.

- Web Browser: Brave
- Local OS: macOS
- Remote OS: DSM 7 (Synology)
- Remote Architecture: amd64
- `code-server --version`: 4.9.0 (linuxserver docker container)

Steps to Reproduce
1. open code-server in browser
2. fill in GUI code 
3. install Python extension
4. create python file
5. can't choose an interpeter
6. install python via terminal in Coder GUI
7. Coder automatically fills in or "press return" after SUDO password prompt for user "abc" resulting in incorrect password response
8. on 2nd\3rd attempt it allows for manual input - PASSWORD_SUDO is not accepted anyway

Expected
sudo password set via variable PASSWORD_SUDO should have been accepted

Actual
terminal shows sudo password is incorrect

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/25886632/224821847-6997381a-bd1d-473d-a663-863765ab8bcb.png">

P.S. I do use container port 8443, as suggested on [linuxserver coder page](https://github.com/linuxserver/docker-code-server#parameters), but it is http.

**Comment by code-asher** [comment]
Interesting, just like you for me the first password attempt auto-submitted but I was able to type my password on the second attempt however for me it accepted the password on the second attempt.

I am not sure what could cause this so someone would need to investigate further.  As a workaround you could try running those commands manually in the terminal?  Might have better luck that way.
